packages/mongodb-runner/src/mongoserver.ts

```diff
@@ -197,10 +197,10 @@ export class MongoServer {
       await once(outStream, 'open');
       stdout.pipe(outStream, { end: false });
       stderr.pipe(outStream, { end: false });
-      Promise.all([once(stdout, 'end'), once(stderr, 'end')]).then(
+      Promise.allSettled([once(stdout, 'end'), once(stderr, 'end')]).then(
         () => outStream.end(),
         () => {
-          /* ignore error */
```

Comment:
is this still not ignoring an error?

Reply:
`.allSettled()` never rejects, hence the change in description here – the behavior is similar (but now we always wait for _both_ stdout and stderr to be closed successfully instead of one of them potentially closing because of an error)